compare/compare.c

```diff
@@ -569,7 +571,7 @@ static int *parse_cmdline(int argc, char *argv[])
 fail:
     free(pos);
     const char *fmt =
-        "Usage: %s ([-i] <input-dir>)+ [-o <output-dir>] [-s <scale:1-8>] [-p <pass-level:0-3>]\n";
+        "Usage: %s ([-i] <input-dir>)+ [-o <output-dir>] [-s <scale:1-8>] [-a <anamorphic-scale:1-8>] [-p <pass-level:0-3>]\n";
```

Comment:
Or maybe `-s` should just accept a pair of integers? Like, `[-s <scale:1-8>[x<scaleY:1-8>]]`?

Reply:
Yes, that variant would be more obvious.

Reply:
Amended to take `-s <scale:1-8>[x<scale:1-8>]` with an expanded usage message.

Reply:
And again to update `compare/README.md`.